scene/gui/tab_bar.cpp

```diff
@@ -1618,6 +1710,8 @@ void TabBar::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_current_tab", "tab_idx"), &TabBar::set_current_tab);
 	ClassDB::bind_method(D_METHOD("get_current_tab"), &TabBar::get_current_tab);
 	ClassDB::bind_method(D_METHOD("get_previous_tab"), &TabBar::get_previous_tab);
+	ClassDB::bind_method(D_METHOD("select_tab_on_left"), &TabBar::select_tab_on_left);
+	ClassDB::bind_method(D_METHOD("select_tab_on_right"), &TabBar::select_tab_on_right);
```

Comment:
These two methods should probably be called `select_next_available` and `select_previous_available`. Current names miss the fact that it can skip tabs or wrap around. And it also won't make sense if we decide to allow for vertical tabs.

Reply:
I originally wanted to use `select_next_tab` and `select_previous_tab`, but there was already member called `previous_tab` for the previously active tab and I couldn't think of any better name. Thanks for the suggestion.